src/components/modals/OperationDetails.js

```diff
@@ -132,7 +133,7 @@ const mapStateToProps = (state, { operationId, accountId, parentId }) => {
     ? confirmationsNbForCurrencySelector(state, { currency: mainCurrency })
     : 0
   let operation = null
-  if (account) {
+  if (account && account.type === 'Account') {
```

Comment:
i'll fix `findOperationInAccount` to work for a token account but it should probably not filter ==='Account' here